### DISCUSSION TITLE
ollama baseURL error (connect timeout=10)

### DISCUSSION THREAD

**Discussion by Aexuss** [langgenius/dify#4433] [Help]
An error cccurred during credentials validation: HTTPConnectionPool(host='192.168.1.100', port=11434): Max retries exceeded with url: /api/chat (Caused by ConnectTimeoutError(<urllib3.connection.HTTPConnection object at Ox7fc799798e20>, 'Connection to 192.168.1.100 timed out. (connect timeout=10)'))

 Windows
Dify is deployed using docker

<img width="457" alt="e2" src="https://github.com/langgenius/dify/assets/169800618/8731d942-79d7-4fc4-ba00-cca30912cbc9">
<img width="1173" alt="e1" src="https://github.com/langgenius/dify/assets/169800618/34c71153-3fe9-4386-b215-85f6d188f3f3">

<img width="1156" alt="e3" src="https://github.com/langgenius/dify/assets/169800618/0ad8f3b9-731a-4727-bfad-f5c78b820e6b">

- Fail to work：

    - http://localhost:11434

    - http://127.0.0.1:11434

    - http://172.17.0.1:11434

**Comment by guang8888** [comment]
do you resolve this error, can you share your resolver
